src/vm/runners/cairo_runner.rs

```diff
@@ -33,7 +33,7 @@ pub struct CairoRunner<'a> {
     program_base: Option<Relocatable>,
     execution_base: Option<Relocatable>,
     initial_ap: Option<Relocatable>,
-    initial_fp: Option<Relocatable>,
+    pub initial_fp: Option<Relocatable>,
```

Comment:
Make a method. We want to expose as little as possible of the actual structures so we can change them later on for refactors, optimizations and, if needed, functionality.
Besides, if at all possible we want to avoid it being written at random moments by downstream as it could lead to inconsistencies.